src/kernels/jupyter/jupyterKernelService.node.ts

```diff
@@ -276,75 +274,35 @@ export class JupyterKernelService implements IJupyterKernelService {
                     );
                     specModel.argv[0] = interpreter.uri.fsPath;
                 }
-                // Get the activated environment variables (as a work around for `conda run` and similar).
-                // This ensures the code runs within the context of an activated environment.
-                const interpreterEnv = await this.activationHelper
```

Comment:
Removed duplicate code in favor of `kernelEnvVars.getEnvironmentVariables`
Now both raw and jupyter kernels (both) use the exact same code to generate env variables for kernels.